tests/test_resource.py

```diff
@@ -295,10 +301,10 @@ def test_options_request(self):
         }
 
 
-class TestPagination:
+class TestPagination:  # pylint: disable=attribute-defined-outside-init, unused-argument
     @patch.object(PaginatedCORSResource, "__init__", return_value=None)
     def setup_method(self, test_method, init_mock) -> None:
-        self.resource = PaginatedCORSResource()
+        self.resource = PaginatedCORSResource()  # pylint: disable=no-value-for-parameter
```

Comment:
Here, I think the code should be improved instead 🤔 

Even simple `self.resource = PaginatedCORSResource({}, [])` should do the thing.